compiler/il/Aliases.cpp

```diff
@@ -142,6 +142,10 @@ OMR::SymbolReference::getUseonlyAliasesBV(TR::SymbolReferenceTable * symRefTab)
             {
             return &symRefTab->aliasBuilder.defaultMethodUseAliases();
             }
+         if (symRefTab->isNonHelper(self(), TR::SymbolReferenceTable::nonNullableArrayNullStoreCheckSymbol))
+            {
+            return &symRefTab->aliasBuilder.defaultMethodUseAliases();
```

Comment:
This seems to make the aliasing conservative (but it is obviously functionally ok). Is this the final state you envision or were you thinking of just delivering the basic support now for this new symbol and look at aliasing refinements for all these helper calls later ?

Reply:
As with the use-def aliases, above, I was actually just following along with the implementations of other non-helper symbols, such as objectEqualityComparisonSymbol, and I wasn't thinking too much about refining the aliasing.  We could certainly do better in terms of the aliasing. . . .  I can open a follow on issue for that as a potential improvement, if you're OK with this for now.

Reply:
I am fine with a follow on issue (see my slightly longer answer to the earlier similar comment at a different spot in the code).